src/io/asyncsocket.c

```diff
@@ -732,10 +851,8 @@ static void listen_setup(MVMThreadContext *tc, uv_loop_t *loop, MVMObject *async
     li->work_idx   = MVM_io_eventloop_add_active_work(tc, async_task);
 
     /* Create and initialize socket and connection, and start listening. */
```

Comment:
Is this comment still correct that it is initializing a socket?

Reply:
No